sigh/lib/assets/resign.sh

```diff
@@ -787,7 +787,7 @@ function resign {
         OLD_BUNDLE_ID="$(PlistBuddy -c "Print :CFBundleIdentifier" "$TEMP_DIR/oldInfo.plist")"
         NEW_BUNDLE_ID="$(bundle_id_for_provison "$NEW_PROVISION")"
         log "Replacing old bundle ID '$OLD_BUNDLE_ID' with new bundle ID '$NEW_BUNDLE_ID' in patched entitlements"
-        sed -i .bak "s/$OLD_BUNDLE_ID/$NEW_BUNDLE_ID/g" "$PATCHED_ENTITLEMENTS"
+        sed -i .bak "s!${OLD_BUNDLE_ID}Enterprise</string>!${NEW_BUNDLE_ID}</string>!g" "$PATCHED_ENTITLEMENTS"
```

Comment:
Enterprise?

Reply:
Oh sorry. I pushed the wrong change. That was just in there for testing.

Reply:
@janpio Correct fix pushed now!